nginx/1.11/README.md

```diff
@@ -0,0 +1,92 @@
+# How to use NGINX on DC/OS
+
+[NGINX](https://www.nginx.com) is a high-performance HTTP server, reverse proxy, and an IMAP/POP3 proxy server. NGINX is known for its high performance, stability, rich feature set, simple configuration, and low resource consumption. DC/OS allows you to quickly configure, install and manage NGINX.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone interested in running a HTTP (proxy) server
+- Scope: Learn how to deploy NGINX service on DC/OS and to serve a static website
+
+## Prerequisites
+
+- A running DC/OS 1.10 cluster with at least 1 node having at least 1 CPUs and 1 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
```

Comment:
Since this is the example for 1.11, you'll want to update the requirement here to say 1.11 and link to the 1.11 documentation link here (same link, just 1.11 instead of 1.10)